### PR TITLE
tidb-lightning: add a new doc about compatibility and scenarios

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#### 场景 2：物理导入模式导入完成后，该表不会有 DML 操作
+
+该场景由于不涉及增量数据写入，因此在 TiDB Lightning 物理导入模式完成数据导入后，对该表执行一次表级别快照备份即可，操作步骤请参考[备份单张表的数据](/br/br-snapshot-manual.md#备份单张表的数据)。
+
+在数据恢复时，对该表的快照数据进行恢复，操作步骤请参考[恢复单张表格](/br/br-snapshot-manual.md#恢复单张表的数据)。
+
+#### 场景 3：物理导入模式导入完成后，该表会执行 DML 操作（不支持）
+
+该场景下，对该表的备份操作，只能在执行全量快照备份和增量备份之间二选一，无法备份并恢复该表的全量快照数据+增量数据。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:48]:
```suggestion
该场景下，对该表的备份操作，只能在执行全量快照备份和日志备份之间二选一，无法备份并恢复该表的全量快照数据+日志备份数据。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### TiDB Lightning 物理导入模式和 TiCDC
+
+该场景短期内无法兼容，因为 TiCDC 也很难追上 TiDB Lightning 物理导入的写入速度，造成集群同步延迟不断增加。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:52]:
```suggestion
该场景短期内无法兼容，因为 TiCDC 很难追上 TiDB Lightning 物理导入的写入速度，造成集群同步延迟不断增加。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+在该场景下，如果上游 TiDB 集群某张表使用 TiDB Lightning 物理导入模式导入数据，而且需要被同步到下游，建议按如下步骤操作：
+
+- 在该场景下，由于开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。
+
+- 使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，这些数据库也支持直接从 Cloud Storage 读取 CSV、Parquet 等文件并写入到数据库。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:58]:
```suggestion
可根据不同的场景如如下操作：

- 场景1：该表不需要被 TiCDC 同步到下游。在该场景下，由于开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。

- 场景2：该表需要被同步到下游。在该场景下，由于上游 TiDB 集群开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。在上游 TiDB 集群把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。之后，使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等文件并写入到数据库。
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+本文档介绍 TiDB Lightning 和 `IMPORT INTO` 与[日志备份](/br/br-pitr-guide.md)、[TiCDC](/ticdc/ticdc-overview.md) 的兼容性，以及某些特殊的使用场景。
+
+## `IMPORT INTO` 和 TiDB Lightning 对比
+
+[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 目前已经集成了 TiDB Lightning 的物理导入模式，但二者还存在一些差异。详情请参见 [`IMPORT INTO` 和 TiDB Lightning 对比](/tidb-lightning/import-into-vs-tidb-lightning.md)
+
+> **注意：**
+>
+> 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 相比，[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 语句可以直接在 TiDB 节点上执行，支持自动化分布式任务调度和 [TiDB 全局排序](/tidb-global-sort.md)，在部署、资源利用率、任务配置便捷性、调用集成便捷性、高可用性和可扩展性等方面都有很大提升。建议在合适的场景下，使用 `IMPORT INTO` 代替 TiDB Lightning。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:16]:
```suggestion
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## `IMPORT INTO` 和 TiDB Lightning 对比
+
+[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 目前已经集成了 TiDB Lightning 的物理导入模式，但二者还存在一些差异。详情请参见 [`IMPORT INTO` 和 TiDB Lightning 对比](/tidb-lightning/import-into-vs-tidb-lightning.md)
+
+## 与日志备份和 TiCDC 的兼容性
+
+- TiDB Lightning 的[逻辑导入模式](/tidb-lightning/tidb-lightning-logical-import-mode.md)与日志备份以及 TiCDC 兼容。
+
+- TiDB Lightning 的[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)与日志备份以及 TiCDC 均不兼容。原因是 TiDB Lightning 物理导入模式或 `IMPORT INTO`，均是将源数据编码后的 KV Pairs 直接 Ingest 到 TiKV，该过程 TiKV 不会产生相应的 Change log，由于没有这部分的 Change log，相关数据无法被日志备份备份，也无法被 TiCDC 复制。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:18]:
```suggestion
- TiDB Lightning 的[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)与日志备份以及 TiCDC 均不兼容。原因是 TiDB Lightning 物理导入模式或 `IMPORT INTO`，均是将源数据编码后的 KV Pairs 直接 Ingest 到 TiKV，该过程 TiKV 不会产生相应的 Change log，由于没有这部分的 Change log，相关数据无法通过日志备份的方式备份，也无法被 TiCDC 复制。
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---
+title: IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼容性
+summary: 了解 IMPORT INTO 和 TiDB Lightning、日志备份、TiCDC 的兼容性及使用场景。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:3]:
```suggestion
summary: 了解 IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼容性及使用场景。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+如果业务对导入性能有要求，且只能使用 TiDB Lightning 物理导入模式的场景，同时这些表还需要备份或者使用 TiCDC 同步到下游，则建议使用以下方案进行处理。
+
+### 和日志备份同时使用
+
+#### 场景 1：物理导入模式的表不需要备份
+
+该场景下，由于开启了 [PITR](/br/br-log-architecture.md#pitr)，因此启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要备份或者[日志备份](/br/br-pitr-guide.md)，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。
+
+#### 场景 2：物理导入模式导入完成后，该表不会有 DML 操作
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:36]:
```suggestion
#### 场景 2：物理导入模式导入完成后，该表不会有新的 DML 操作
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+该场景由于不涉及增量数据写入，因此在 TiDB Lightning 物理导入模式完成数据导入后，对该表执行一次表级别快照备份即可，操作步骤请参考[备份单张表的数据](/br/br-snapshot-manual.md#备份单张表的数据)。
+
+在数据恢复时，对该表的快照数据进行恢复，操作步骤请参考[恢复单张表格](/br/br-snapshot-manual.md#恢复单张表的数据)。
+
+#### 场景 3：物理导入模式导入完成后，该表会执行 DML 操作（不支持）
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:42]:
```suggestion
#### 场景 3：物理导入模式导入完成后，该表会执行新的 DML 操作（不支持）
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+该场景下，对该表的备份操作，只能在执行[全量快照备份](/br/br-snapshot-guide.md)和[日志备份](/br/br-pitr-guide.md)之间二选一，无法备份并恢复该表的全量快照数据+日志备份数据。
+
+### 和 TiCDC 同时使用
+
+该场景短期内无法兼容，因为 TiCDC 很难追上 TiDB Lightning 物理导入的写入速度，造成集群同步延迟不断增加。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:48]:
```suggestion
该场景短期内无法兼容，因为 TiCDC 很难追上 TiDB Lightning 物理导入的写入速度，可能造成集群同步延迟不断增加。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+可根据如下不同的场景进行操作：
+
+- 场景 1：该表不需要被 TiCDC 同步到下游。在该场景下，由于开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:52]:
```suggestion
- 场景 1：该表不需要被 TiCDC 同步到下游。在该场景下，如果开启了 TiCDC 同步任务，启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动导入任务即可。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+- 场景 2：该表需要被 TiCDC 同步到下游。在该场景下，由于上游 TiDB 集群开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。你需要在上游 TiDB 集群把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。之后，使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等文件并写入到数据库。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:54]:
```suggestion
- 场景 2：该表需要被 TiCDC 同步到下游。在该场景下，由于上游 TiDB 集群开启了 TiCDC 同步任务，因此启动 TiDB Lightning 后兼容性检查会报错。你需要在上游 TiDB 集群把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动导入任务即可。上游 TiDB 集群的导入任务完成后，再使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等格式的文件并写入到数据库。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+## `IMPORT INTO` 的使用场景
+
+本节介绍 `IMPORT INTO` 与[日志备份](/br/br-pitr-guide.md)和 [TiCDC](/ticdc/ticdc-overview.md) 同时使用时的操作方法。
+
+### 和日志备份同时使用
+
+该场景下，由于开启了 [PITR](/br/br-log-architecture.md#pitr)，因此提交 `IMPORT INTO` SQL 后兼容性检查会报错。如果你确定这些表不需要备份或者[日志备份](/br/br-pitr-guide.md)，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:62]:
```suggestion
该场景下，如果开启了 [PITR](/br/br-log-architecture.md#pitr)，提交 `IMPORT INTO` SQL 语句后兼容性检查会报错。如果你确定这些表不需要备份或者[日志备份](/br/br-pitr-guide.md)，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+### 和 TiCDC 同时使用
+
+在该场景下，由于开启了 TiCDC Changefeed，因此提交 `IMPORT INTO` SQL 语句后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:66]:
```suggestion
在该场景下，如果开启了 TiCDC Changefeed，提交 `IMPORT INTO` SQL 语句后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+
+- 场景 1：该表不需要被 TiCDC 同步到下游。在该场景下，由于开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。
+
+- 场景 2：该表需要被 TiCDC 同步到下游。在该场景下，由于上游 TiDB 集群开启了 TiCDC Changefeed，因此启动 TiDB Lightning 后兼容性检查会报错。你需要在上游 TiDB 集群把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动任务即可。之后，使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等文件并写入到数据库。
+
+## `IMPORT INTO` 的使用场景
+
+本节介绍 `IMPORT INTO` 与[日志备份](/br/br-pitr-guide.md)和 [TiCDC](/ticdc/ticdc-overview.md) 同时使用时的操作方法。
+
+### 和日志备份同时使用
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:60]: 这里虽然只写了一个场景，但是 TiDB Lightning 物理导入模式的使用场景 > 和日志备份同时使用 里的场景 2 和场景 3 看起来对 `IMPORT INTO` 也适用。

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+
+该场景下，由于开启了 [PITR](/br/br-log-architecture.md#pitr)，因此提交 `IMPORT INTO` SQL 后兼容性检查会报错。如果你确定这些表不需要备份或者[日志备份](/br/br-pitr-guide.md)，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
+
+### 和 TiCDC 同时使用
+
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:65]: 这里虽然只写了一个场景，但是 TiDB Lightning 物理导入模式的使用场景 > TiCDC 同时使用 里的场景 2 看起来对 `IMPORT INTO` 也适用，只需要改成 “你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可”

**Review by qiancai — COMMENTED** [review]
@hfxsd I've added some comments. PTAL.

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md (hunk 14 above)
Comment (Frank945946) [reply tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:62]: 是的

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md (hunk 15 above)
Comment (Frank945946) [reply tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:67]: 如果是分开写，建议写全一些，如果是 import into 和 Lightning 合并到一块，那可以简写。

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+- 场景 1：该表不需要被 TiCDC 同步到下游。
+
+    在该场景下，如果开启了 TiCDC Changefeed，提交 `IMPORT INTO` SQL 语句后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可，这样数据导入任务会忽略该兼容性检查，直接导入数据。
+
+- 场景 2：该表需要被 TiCDC 同步到下游。
+
+    在该场景下，如果上游 TiDB 集群开启了 TiCDC 同步任务，提交 `IMPORT INTO` SQL 语句后兼容性检查会报错。你需要在该 SQL 的 [`WithOptions`](/sql-statements/sql-statement-import-into.md#withoptions) 里带上参数 `DISABLE_PRECHECK`（从 v8.0.0 版本引入）重新提交即可。
+
+    上游 TiDB 集群的导入任务完成后，再使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等格式的文件并写入到数据库。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:100]:
```suggestion
    上游 TiDB 集群的导入任务完成后，再使用 `IMPORT INTO` 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等格式的文件并写入到数据库。
```

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 17]
```diff
@@ -0,0 +1,100 @@
...
+
+- 场景 1：该表不需要被 TiCDC 同步到下游。
+
+    在该场景下，如果开启了 TiCDC 同步任务，启动 TiDB Lightning 后兼容性检查会报错。如果你确定这些表不需要被 TiCDC 同步，你可以把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动导入任务即可。
+
+- 场景 2：该表需要被 TiCDC 同步到下游。
+
+    在该场景下，由于上游 TiDB 集群开启了 TiCDC 同步任务，因此启动 TiDB Lightning 后兼容性检查会报错。你需要在上游 TiDB 集群把 [TiDB Lightning 配置文件](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-任务配置)中的 `Lightning.check-requirements` 参数改成 `false`，然后重新启动导入任务即可。
+
+    上游 TiDB 集群的导入任务完成后，再使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、Parquet 等格式的文件并写入到数据库。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:64]:
```suggestion
    上游 TiDB 集群的导入任务完成后，再使用 TiDB Lightning 在下游 TiDB 集群也导入一份同样的数据。如果下游是 Redshift、Snowflake 等数据库，可直接让这些数据库从 Cloud Storage 读取 CSV、SQL、Parquet 等格式的文件并写入到数据库。
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md (hunk 15 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:89]: 都加上了

File: tidb-lightning/tidb-lightning-compatibility-and-scenarios.md [hunk 18]
```diff
@@ -0,0 +1,100 @@
+---
+title: IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼容性
+summary: 了解 IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼容性及使用场景。
+---
+
+# IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼容性
+
+本文档介绍 TiDB Lightning 和 `IMPORT INTO` 与[日志备份](/br/br-pitr-guide.md)、[TiCDC](/ticdc/ticdc-overview.md) 的兼容性，以及某些特殊的使用场景。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-compatibility-and-scenarios.md:8]:
```suggestion
本文档介绍 TiDB Lightning 和 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 与[日志备份](/br/br-pitr-guide.md)、[TiCDC](/ticdc/ticdc-overview.md) 的兼容性，以及某些特殊的使用场景。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**